app/controllers/discourse_subscriptions/admin/plans_controller.rb

```diff
@@ -9,8 +9,15 @@ class PlansController < ::Admin::AdminController
 
       def index
         begin
-          plans = ::Stripe::Price.list(product_params)
+          plans = ::Stripe::Price.list({ limit: 100 })
+          products = ::Stripe::Product.list({ limit: 100 })
 
+          plans.data.each do |plan|
+            products.data.each do |product|
+              plan.product_name = product.name if plan.product == product.id
+            end
+          end
+          puts products
```

Comment:
A nested loop in a controller is generally inefficient. You can do something like this instead:
```suggestion
          products_by_id = products.data.index_by(&:id)
          
          plans.data.each do |plan|
            plan.product_name = products_by_id[plan.product]&.name
         end
```